app/Controllers/PrestitiController.php

```
@@ -168,6 +168,16 @@ public function createForm(Request $request, Response $response, mysqli $db): Re
             }
         }
 
+        // Prefill delle date nel timezone APPLICATIVO e con la durata configurata:
+        // il vecchio date('Y-m-d') (TZ processo, spesso UTC) mostrava "ieri" dopo
+        // mezzanotte, e il '+1 month' della view divergeva dal default server (30gg).
+        $defaultDataPrestito = \App\Support\DateHelper::today();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\DateHelper' in method 'createForm'. (undefined)

(StaticAccess)